tests/testsuite/required_features.rs

```diff
@@ -399,29 +419,54 @@ fn test_multiple_required_features() {
         .build();
 
     p.cargo("test")
-        .with_stderr(
-            "\
-[COMPILING] foo v0.0.1 ([CWD])
-[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [..]
-[RUNNING] [..] (target/debug/deps/foo_2-[..][EXE])",
-        )
-        .with_stdout_contains("test test ... ok")
+        .with_stderr_data(str![[r#"
+[COMPILING] foo v0.0.1 ([ROOT]/foo)
+[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
+[RUNNING] tests/foo_2.rs (target/debug/deps/foo_2-[HASH][EXE])
+
+"#]])
+        .with_stdout_data(str![[r#"
+
+running 1 test
+test test ... ok
+
+test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in [ELAPSED]s
+
+
+"#]])
         .run();
 
     p.cargo("test --features c")
-        .with_stderr(
-            "\
-[COMPILING] foo v0.0.1 ([CWD])
-[FINISHED] `test` profile [unoptimized + debuginfo] target(s) in [..]
-[RUNNING] [..] (target/debug/deps/foo_1-[..][EXE])
-[RUNNING] [..] (target/debug/deps/foo_2-[..][EXE])",
-        )
-        .with_stdout_contains_n("test test ... ok", 2)
```

Comment:
The `with_stdout_contains_n` function should be analogous to `with_stdout_contains` where the expected output appears n times, and the tracking issue doesn't mention this function.